### ISSUE TITLE
__enter__() should return self

### ISSUE THREAD

**Issue by dmitriykovalev** [vsergeev/python-periphery#17]
Current implementation (e.g. in gpio.py):
```python
def __enter__(self):
    pass
```
Ideally should be
```python
def __enter__(self):
    return self
```
to make the following example working:
```python
with GPIO(10, "out") as pin:
     pin.write(True)
```

**Comment by ukleinek** [comment]
This is fixed since a05c229d98ad0460ec2221f25832f3cccf489fc1. There is only one instance left
in `tests/` which shouldn't be relevant to users of periphery.

**Comment by vsergeev** [comment]
Thanks for pointing that out @ukleinek. I pushed a commit to fix that one as well. As @ukleinek said, these are now fixed in `master` and will be included in the next release.
